exercises/luhn/luhn_test.exs

```diff
@@ -0,0 +1,44 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("luhn.exs")
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule LuhnTest do
+  use ExUnit.Case
+
+  test "checksum" do
+    assert Luhn.checksum("4913") == 22
+  end
+
+  @tag :pending
+  test "checksum again" do
```

Comment:
Should we move these "again" tests into a single test or there is some rule in exercism that we should not have multiple assertions per test? I'm asking this because we are not testing a new scenario here, so it would make sense to have just a `checksum` test with both assertions. What do you think? :)